lib-multisrc/mangabox/build.gradle.kts

```
@@ -2,4 +2,4 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 5
+baseVersionCode = 6
```

Don't bump if you are only changing the visibility of functions or whatnot.
```suggestion
baseVersionCode = 5
```